OKR.md

```diff
@@ -2,6 +2,32 @@
 
 We try to frame our ongoing work using a process based on quarterly Objectives and Key Results (OKRs). Objectives reflect outcomes that are challenging, but realistic. Results are tangible and measurable.
 
+## 2018 Q3
+
+**The Connectivity Magic story is figured out and solved**
+
+- `P1` - @diasdavid -     A js-ipfs Daemon will always find other IPFS Daemons (js & go) in the LAN (MulticastDNS)
+- `P0` - @diasdavid -     The DHT is enabled by default in js-ipfs
+- `P1` - @diasdavid -     Circuit Relay Interop tests are finalized and merged (depends on go-ipfs 0.4.16)
+- `P0` - ???        -     A js-ipfs node (Browser & Desktop) always finds files that are on the Gateways
+
+**Daemon is as reliable as it can be**
+
+- `P0` - @alanshaw -      A Daemon runs for 10 days without a crash 
+- `P2` - ??? -            One or more js-ipfs daemons are part of the Bootsrapper nodes 
+- `P0` - @hugomrdias -    js-ipfs can handle 1TB of data, both Node.js and Browser
+- `P1` - @achingbrain -   npm on IPFS over js-ipfs is a sound way to install deps and gets used for CI
+- `P?` - ??? -            <Something on integration testing between Desktop, Browser and go-ipfs nodes with multiple network configs>
+
+**New user/contributor experience is extraordinary**
+
+- `P0` - @diasdavid -     js.ipfs.io is finished and published 
+- `P0` - @diasdavid -     Onboard 2 new full time contributors 
+- `P2` - ???        -     Complete the Awesome Endeavour, Documentation
+- `P2` - @alanshaw  -     Cut CI testing time in js-ipfs and js-ipfs-api by 50% (2x) 
+
```

Comment:
It's time to wrap this up. 

@alanshaw @achingbrain @hugomrdias, ideally, each should have at least 4 KRs with two of them being a P0. Right now the KR distribution is kind of unbalanced and it is not giving you a set of problems to focus.

Also, I'm not confident that the list is as polished as it can be for the Objectives we have. Now that the Dev Meetings happened and your understanding of the whole project expanded vastly, I want you to go through these again and come up with new proposal. Please put this as your top priority until we freeze KRs next Wednesday.

Reply:
I didn't appreciate the time frame on the switch to base32 until the developer meetings. I'm happy to spearhead the development work that needs to happen on the JS side of things for this. BTW this is a P0 for In Web Browsers OKRs. I'm also going to continue to be involved with the WebUI rebuild as part of the GUI WG.